doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2409,3 +2409,55 @@ setting indicates embargoes are not supported. A value of -1 allows embargoes of
 can enter for an embargo end date. This limit will be enforced in the popup dialog in which users enter the embargo date. For example, to set a two year maximum:
 
 ``curl -X PUT -d 24 http://localhost:8080/api/admin/settings/:MaxEmbargoDurationInMonths``
+
+:DataverseMetadataValidatorScript
```

Comment:
Just a thought, you could use the word "collection" for the dataverse settings, such as `:CollectionMetadataValidatorScript`.

Reply:
These existing settings have been in use for a year in prod. I'm not messing with them. Legacy.